README.rst

```diff
@@ -212,6 +212,7 @@ In order to run the unit tests and to get a coverage report, five additional Pyt
   pip3 install --user mpi4py
   pip3 install --user tblib
   pip3 install --user pytest
+  pip3 install --user astunparse
```

Comment:
We now have six additional packages, not five anymore. Hence in the sentence just above this list I suggest that we change the word "five" with "some" or "a few", so that we will not need to update it in the future.

Reply:
Fixed in 92deae4